examples/user_guide/Links.ipynb

```diff
@@ -192,21 +273,49 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "watcher = widget_toggle.param.watch(lambda change: setattr(widget_text, 'disabled', not change.new), 'active' )"
+    "#toggle.param.unwatch(watcher)"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Now cliking on the toggle button enable or disable the text input"
+    "## Linking objects in JS\n",
+    "\n",
+    "Linking objects in Python is often very convenient, because it allows writing code entirely in Python. However, it also requires a live Python kernel. If instead we want a static example (e.g. on a website or in an email) to have custom interactivity, or we simply want to avoid the overhead of having to call back into Python, we can define links in JavaScript.\n",
+    "\n",
+    "### Linking Panel components\n",
+    "\n",
+    "To begin with let us see how we can express simple links between standard Panel components before digging into some more specific and complex examples.\n",
+    "\n",
+    "#### Linking model properties\n",
+    "\n",
+    "Let us start by rehashing the example from above, linking the ``value`` of the ``TextInput`` widget to the ``object`` property of the ``Markdown`` pane:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "markdown = pn.pane.Markdown('Markdown display')\n",
+    "text_input = pn.widgets.TextInput(value=markdown.object)\n",
+    "\n",
+    "link = text_input.jslink(markdown, value='object')\n",
+    "\n",
+    "pn.Row(text_input, markdown)"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "`param.watch` return a reference to the watcher which can be used to remove it."
+    "As you can see the signature is identical to the ``link`` example above, but here Panel translates the specification into a JS code snippet which syncs the properties on the underlying Bokeh properties. But now if you edit the widget and press Return, the Markdown display will automatically update even in a static HTML web page.\n",
+    "\n",
+    "#### Linking using custom JS code\n",
+    "\n",
+    "Since everything happens in JS for a `jslink`, we can't provide a Python callback. Instead, we can define a JS code snippet, which is executed when a property changes. E.g. we can define a  little code snippet which adds HTML bold tags (``<b>``) around the text before setting it on the target. The code argument should map from the parameter/propery on the source object to the JS code snippet to execute:"
```

Comment:
```suggestion
    "Since everything happens in JS for a `jslink`, we can't provide a Python callback. Instead, we can define a JS code snippet, which is executed when a property changes. E.g. we can define a little code snippet which adds HTML bold tags (``<b>``) around the text before setting it on the target. The code argument should map from the parameter/property on the source object to the JS code snippet to execute:"
```